src/DocBlock/Description.php

```diff
@@ -66,6 +66,15 @@ public function __construct(string $bodyTemplate, array $tags = [])
         $this->bodyTemplate = $bodyTemplate;
         $this->tags = $tags;
     }
+    
+    /**
+     * Returns the body template
+     * @return string
```

Comment:
Small note: can you add a whiteline between the summary and the tags for consistency?